sympy/core/tests/test_sympify.py

```diff
@@ -161,11 +161,19 @@ def test_sympify_bool():
 def test_sympyify_iterables():
     ans = [Rational(3, 10), Rational(1, 5)]
     assert sympify(['.3', '.2'], rational=True) == ans
-    assert sympify(tuple(['.3', '.2']), rational=True) == Tuple(*ans)
     assert sympify(dict(x=0, y=1)) == {x: 0, y: 1}
     assert sympify(['1', '2', ['3', '4']]) == [S(1), S(2), [S(3), S(4)]]
 
 
+@XFAIL
+def test_issue_16772():
```

Comment:
add `ans = [Rational(3, 10), Rational(1, 5)]` and perhaps remove the XFAIL...this is a useful 'feature': if you want an iterable trversed with the flag you have to pass it as a list.